phpBB/config/production/container/services.yml

```diff
@@ -1,2 +1,8 @@
 imports:
     - { resource: ../../default/container/services.yml }
+
+services:
```

Comment:
Reminder: Move this to a separate file, and import that instead.

Although if anyone has a better idea for handling this, that is certainly welcomed.